bower.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.3.0",
   "homepage": "https://github.com/Squareknot/marionette.state",
   "authors": [
-    "ianmstew <ianmstew@gmail.com>"
+    "ianmstew <ianmstew@gmail.com>",
+    "yurkaninryan <yurkaninryan@gmail.com>"
```

Comment:
Haha, I'm not sure that [package.json authors](https://github.com/marionettejs/backbone.marionette/blob/master/package.json#L29-L33) are the same as core contributors.  I would say that the minimum bar is that everyone in authors are those with NPM (publish) access.  :p